lnrpc/routerrpc/router.proto

```diff
@@ -467,6 +467,66 @@ message SetMissionControlConfigResponse {
 }
 
 message MissionControlConfig {
+    /*
+    The maximum number of payment results that mission control will store.
+    */
+    uint32 maximum_payment_results = 4;
+
+    /*
+    The minimum time that must have passed since the previously recorded failure
+    before we raise the failure amount.
+    */
+    uint64 minimum_failure_relax_interval = 5;
+
+    enum ProbabilityModel {
+        UNKNOWN = 0;
+        APRIORI = 1;
+        BIMODAL = 2;
+    }
+
+    /*
+    ProbabilityModel defines which probability estimator should be used in
+    pathfinding.
+    */
+    ProbabilityModel Model = 6;
+
+    /*
+    EstimatorConfig is populated dependent on the estimator type.
+    */
+    oneof EstimatorConfig {
```

Comment:
Why make it a breaking change and not follow the usual procedure of deprecating first?

Reply:
Does any major application/wallet actually _use_ this value? We added it originally for some experiments that were never actually even deployed fwiw. I think being able to switch algos on the fly is really valuable for future tuning and experiments.

Reply:
I think keeping things backwards compatible on the rpc level doesn't need to rule out the ability to switch algos on the fly. But yes, if no one uses this, there is no point in keeping the old.

Reply:
As a general rule we should never remove neither change the fields already defined in an older released version of our protos files. We should simply mark the fields (or methods/services) as [deprecated](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/lightning.proto#L4111) and add new ones for the new features.

If it is true that nobody was building on this we can live with it.